modules/runners/main.tf

```
@@ -102,7 +102,8 @@ resource "aws_ssm_parameter" "runner_ami_id" {
   tags = merge(
     local.tags,
     {
-      "ghr:ami_name" = data.aws_ami.runner.name
+      # Remove parentheses from AMI name to comply with SSM tag constraints
```

Assuming you mean AWS tag contraints in general, right?
```suggestion
      # Remove parentheses from AMI name to comply with AWS tag constraints
```